paasta_tools/utils.py

```diff
@@ -659,6 +715,40 @@ def get_fsm_deploy_pipeline(self):
             raise PaastaNotConfiguredError(
                 'Could not find fsm_deploy_pipeline in configuration directory: %s' % self.directory)
 
+    def get_log_writer(self):
+        """Get the log_writer configuration out of global paasta config
+
+        :returns: The log_writer dictionary.
+        """
+        try:
+            return self['log_writer']
+        except KeyError:
+            raise PaastaNotConfiguredError('Could not find log_writer in configuration directory: %s' % self.directory)
```

Comment:
I remember you mentioning this before, but are you really tied to it?

Seems totally legit to me that this would default to say the "stderr" writer, and then you won't have to patch all the things under test?